### DISCUSSION TITLE
Export, display and modify `.ipynb`

### DISCUSSION THREAD

**Discussion by mreboud** [eos/eos#471] [Ideas]
I am not a big fan of how our examples are shared in the github repository and the fact that everything needs to be rewritten in the documentation. Could we try something like https://jupytext.readthedocs.io to improve the handling of these `.ipynb` files?

**Comment by dvandyk** [comment]
I like the idea very much. The problem I see is that our documentation uses ReStructuredText (RST), and not Markdown.

**Reply by dvandyk** [reply]
Seems that @mreboud solved the RST problem :-) The new notebook-based documentation is being built as I write this...
